doc/input-output-reference/src/input-for-output.tex

```diff
@@ -654,24 +701,28 @@ \subsection{Output:EnvironmentalImpactFactors}\label{outputenvironmentalimpactfa
 The energy is obtained internally from the following facility meters:
 
 \begin{longtable}[c]{@{}ll@{}}
-\toprule
-%Electricity:Facility & Diesel:Facility \tabularnewline
-%\midrule
-%\endfirsthead
-%
-%\toprule
-%Electricity:Facility & Diesel:Facility \tabularnewline
-%\midrule
-%\endhead
-%
-Electricity:Facility & Diesel:Facility \tabularnewline
-DistrictCooling:Facility & DistrictHeatingWater:Facility \tabularnewline
-NaturalGas:Facility & Gasoline:Facility \tabularnewline
-Coal:Facility & FuelOilNo1:Facility \tabularnewline
-FuelOilNo2:Facility & Propane:Facility \tabularnewline
-ElectricityProduced:Facility & Steam:Facility \tabularnewline
-OtherFuel1:Facility & OtherFuel2:Facility \tabularnewline
-\bottomrule
+	\caption{Facility Meters \label{table:facility-meters}} \tabularnewline
+	\toprule
+	%Electricity:Facility & Diesel:Facility \tabularnewline
+	Facility Meters & \tabularnewline
+	\midrule
+	\endfirsthead
+
+	\caption[]{Facility Meters} \tabularnewline
+	\toprule
+	%Electricity:Facility & Diesel:Facility \tabularnewline
+	Facility Meters & \tabularnewline
+	\midrule
+	\endhead
```

Comment:
I'm not actually even sure what the second caption command does.  Overwrites the first?

Reply:
@Myoldmopar Actually I had the same doubt when working in a previous PR a while ago--I think I had removed one of then at that time. However, after some digging I found this is a structure of LaTeX `longtable` package that allows the first page of the table to have a different caption from the rest pages of the table (assuming that it could be quite common that a "long" table spans across multiple pages). So this seems to be a reasonable set up use two `caption` placing for this need. In most of the tables, we may not really have this need but it might still ok to keep the structure there. See the doc at https://texdoc.org/serve/longtable.pdf/0 for further references.